### DISCUSSION TITLE
Truncate question

### DISCUSSION THREAD

**Discussion by cdpestitschek** [twilio-labs/paste#2953] [General] [answered]
Hi,
I would like to know if in the case of the ellipse. Is the full text never revealed? Isn't there a way in which the user could read it in its entirety? Eg. having a tooltip on hover.
![Group 1398](https://user-images.githubusercontent.com/100429369/213163775-7a4034b9-6da3-43df-a438-f5d50168abd8.png)

I couldn't find any detail in the description of the component at:
https://paste.twilio.design/components/truncate/

I would like to know because I will have several charts, labels and tables where possibly the truncate happens frequently.

Looking forward to hear back

**Comment by loreina** [comment, marked as answer]
Hi @cdpestitschek,

Our Truncate component uses the browser tooltip so on hover, and you can identify what text will appear on hover using the `title` prop. The string in the `title` prop will then appear on hover like in the example below:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/23108291/213273175-eec6910d-7b62-4590-917a-996466e61c62.png">

Let us know if you have any other questions!

**Reply by cdpestitschek** [reply]
Amazing! Thanks @loreina.
